airbyte-integrations/connectors/source-jdbc/src/main/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSource.java

```diff
@@ -49,15 +55,18 @@
   private static final Logger LOGGER = LoggerFactory.getLogger(AbstractJdbcSource.class);
 
   private static final String JDBC_COLUMN_DATABASE_NAME = "TABLE_CAT";
-  private static final String JDBC_COLUMN_SCHEMA_NAME = "TABLE_SCHEM";
-  private static final String JDBC_COLUMN_TABLE_NAME = "TABLE_NAME";
-  private static final String JDBC_COLUMN_COLUMN_NAME = "COLUMN_NAME";
+  protected static final String JDBC_COLUMN_SCHEMA_NAME = "TABLE_SCHEM";
+  protected static final String JDBC_COLUMN_TABLE_NAME = "TABLE_NAME";
+  protected static final String JDBC_COLUMN_COLUMN_NAME = "COLUMN_NAME";
   private static final String JDBC_COLUMN_DATA_TYPE = "DATA_TYPE";
+  protected static final String GRANTEE = "GRANTEE";
+  protected static final String PRIVILEGE = "PRIVILEGE";
 
   private static final String INTERNAL_SCHEMA_NAME = "schemaName";
   private static final String INTERNAL_TABLE_NAME = "tableName";
   private static final String INTERNAL_COLUMN_NAME = "columnName";
   private static final String INTERNAL_COLUMN_TYPE = "columnType";
+  protected static final String BLANK_STRING = "";
```

Comment:
We can use StringUtils.EMPTY instead of adding new var

Reply:
Fixed. Thanks.